cluster/examples/kubernetes/ceph/filesystem-ec.yaml

```diff
@@ -21,6 +21,9 @@ spec:
     - erasureCoded:
         dataChunks: 2
         codingChunks: 1
+      # Inline compression mode for the data pool
```

Comment:
nit: They can refer to the docs for more links so no need to include this comment in all the yamls. It doesn't hurt though if you want to keep it.

Reply:
Left one instance of the doc link in each of Block, CephFS, RGW instances and removed the rest.